apis/v1alpha1/ansibleRun_types.go

```diff
@@ -40,6 +40,10 @@ type AnsibleRunParameters struct {
 	// +optional
 	Inventories []Inventory `json:"inventories"`
 
+	// This sets the Inventory to executable for use by ansible.builtin.script plugin
+	// +optional
```

Comment:
Why is it used by `ansible.builtin.script` plugin

Reply:
ansible can use python scripts to build inventories using this plugin instead of them being defined statically. https://docs.ansible.com/ansible/latest/inventory_guide/intro_dynamic_inventory.html